app/models/miq_request_workflow.rb

```diff
@@ -780,8 +737,6 @@ def set_default_user_info
   end
 
   def set_request_values(values)
-    # Ensure that tags selected in the pre-dialog get applied to the request
-    values[:vm_tags] = (values[:vm_tags].to_miq_a + @values[:pre_dialog_vm_tags]).uniq unless @values.nil? || @values[:pre_dialog_vm_tags].blank?
```

Comment:
Note: This line was removed because it is already handled in `make_request` from the logic moved from `update_request`.  Both `create_request` and `update_request` were calling it before but in slightly different ways.